### PR TITLE
This add default icon if the image is not avaliable

### PR DESCRIPTION
**Pull request by priyank-p**
Fixes: https://github.com/zulip/zulip-electron/issues/356

### CODE REVIEW

**Comment by akashnimare** [comment]
I think it would be great if we can handle this in our `domain-util` since all the logic for domain/server/icon handling lies there.

**Comment by priyank-p** [comment]
Yeah, i will do so, i did not know where to put so i put it here.

**Comment by priyank-p** [comment]
@akashnimare rather leave it where it right now because `domain-util.js` solely handles downloading icon and add it to db. So we should leave it where it is now since this is where all the event listeners are handled.

**Comment by priyank-p** [comment]
FYI, the tests are failing because, PR #353 is not merged yet.

**Comment by akashnimare** [comment]
@cPhost can you confirm, if it fixes the issue? I still see the broken image icon.

File: app/renderer/js/main.js [hunk 1]
```diff
@@ -149,6 +149,11 @@ class ServerManagerView {
 			this.openSettings('General');
 		});
 
+		const $serverImg = document.querySelector('.server-icons');
+		$serverImg.addEventListener('error', () => {
+			$serverImg.src = 'img/icon.png';
```
Comment (akashnimare) [inline app/renderer/js/main.js]: I don't think the path `img/icon.png` will work here. Try -
```
const APP_ICON = path.join(__dirname, '/img/icon.png');
```

**Comment by akashnimare** [comment]
I should mention that the problem occurs sometimes when we reload the app. In the image below you can see the broken server image but if we do insect and check the src the image loads up successfully. Also, the same image is being used in the server section which surprisingly works correctly -

image

**Comment by akashnimare** [comment]
Merged, thanks @cPhost.

**Pull request merged**